classes/controller/action.php

```diff
@@ -0,0 +1,10 @@
+<?php 
+
+namespace mod_hsuforum\controller;
+
+interface action {
+	public function get_action($id);
+	public function set_action($id);
+	public function delete_action($id);
+	public function render_action();
```

Comment:
Noticed this one is the only one without $id in the params. This intentional?

Reply:
I added a $data variable to it, I had not moved the SQL / HTML return to the class that overrides the interface just yet.